Recruit-Training/Embedded-Recruit-Training/Core/Src/BlueLedToggleTask.c

```diff
@@ -0,0 +1,55 @@
+#include "BlueLedToggleTask.h"
+
+static const uint32_t BLUE_LED_STATUS_STDID = 0xCCC;
+static const uint32_t BLUE_LED_TOGGLE_FREQ = 1000; 
+
+void blueLedToggleTask(void const* arg)
+{
+    //One time osDelayUntil initialization
+    uint32_t prevWakeTime = osKernelSysTick();
+
+    osMutexId_t* canMutex = (osMutexId_t*)arg; // Get mutex that was passed as an argument
+
+    for (;;)
+    {
+        //TODO: Add BLUE_LED_TOGGLE_FREQ to prevWakeTime
+       
+        prevWakeTime += BLUE_LED_TOGGLE_FREQ; // this increments blue toggle with the +=
+        osDelayUntil(prevWakeTime);
+        
+        //TODO: Check blue toggle flag and toggle blue LED
+        if(BlueToggleLED)
+        {
+            HAL_GPIO_TogglePin(LED_RED_GPIO_Port, LED_RED_Pin);
+            
+        }
+
+       uint8_t pinState_of_blue_LED = HAL_GPIO_ReadPin(LED_RED_GPIO_Port, LED_RED_Pin);
```

Comment:
For C, usually we name our variables in `camelCase` instead of using `_`. That notation is more common in Python.